src/collectd.conf.pod

```diff
@@ -8300,6 +8300,13 @@ Synopsis:
      CollectVCL         false
      CollectVSM         false
      CollectWorkers     false
+     CollectLcok        false
```

Comment:
Each of the new config options still needs its own section below. Search for `CollectBan` for a good example.